src/entities/institute/institute.resolver.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Args,
+  Mutation,
+  Query,
+  Resolver,
+  ResolveField,
+  Parent,
+  ID,
+} from '@nestjs/graphql'
+import { Model, Types } from 'mongoose'
+
+// Pagination
+import { PaginationArgs } from '../../utils/Pagination'
+
+import {
+  Institute,
+  InstituteDocument,
+  PaginatedInstitutes,
+} from '../institute/institute.model'
+import { InstituteService } from './institute.service'
+import { SchoolService } from '../school/school.service'
+import { SubjectService } from '../subject/subject.service'
+import { FollowerService } from '../follower/follower.service'
+import {
+  CreateInstituteInput,
+  ListInstituteInput,
+  UpdateInstituteInput,
+} from './institute.inputs'
+
+@Resolver(() => Institute)
+export class InstituteResolver {
+  constructor(
+    private service: InstituteService,
+    private subjectService: SubjectService,
+    private followerService: FollowerService,
+  ) {}
+
+  @Query(() => Institute)
+  async institute(@Args('_id', { type: () => ID }) _id: Types.ObjectId) {
```

Comment:
@frankmangone I'd replace all "_id" with "id" and make a migration, it can be done later though

Reply:
This has been assigned to issue #25